### ISSUE TITLE
input image size for SqueezeNet

### ISSUE THREAD

**Issue by xuewengeophysics** [osmr/imgclsmob#35]
Hi, thanks for the great repo.
For SqueezeNet, the input image size may be 227x227 instead of 224x224, otherwise it will not match the output size.

----------------------
Xue Wen
Shenzhen, China

**Comment by osmr** [comment]
Could you show a script reproducing the problem?

**Comment by xuewengeophysics** [comment]
> Could you show a script reproducing the problem?

https://github.com/osmr/imgclsmob/blob/master/tensorflow_/tensorflowcv/models/squeezenet.py
![image](https://user-images.githubusercontent.com/8676098/65087874-e25f1400-d9e9-11e9-8bf1-e88c494e0022.png)

no padding
![image](https://user-images.githubusercontent.com/8676098/65087959-379b2580-d9ea-11e9-903b-788b88c5bed8.png)

(224 + 2*0 - 7) / 2 + 1 = 109
![image](https://user-images.githubusercontent.com/8676098/65088211-125ae700-d9eb-11e9-9de2-dda6d02b0c36.png)

**Comment by osmr** [comment]
Yes, this is an iron argument. So my implementation does not correspond to the paper. I don’t remember the reason for this. Maybe because all the popular implementations of the SqueezeNet on Gluon and PyTorch don't correspond to the paper.

**Comment by osmr** [comment]
@xuewengeophysics Agree that the top row of the table with the input size 224x224x3 looks a bit provocative:) In addition, the implementation of the network on an alternative framework is always a small compromise. Obviously, the input 227x227 is a unique feature of the SqueezeNet, but it does not determine its speed and accuracy properties.
